src/Frontend/Modules/Mailmotor/Layout/Widgets/Subscribe.html.twig

```diff
@@ -9,7 +9,7 @@
           {{ form_row(form.email) }}
         </p>
 
-        {{ form_rest(form) }}
+        {{ form_row(form.interests) }}
```

Comment:
There are cases when the interests aren't added, this will now give an error in those cases.
You need to check if the field exists